src/cpu/x64/jit_brgemm_conv.cpp

```diff
@@ -834,8 +834,11 @@ inline int brgemm_convolution_fwd_t<isa>::get_comp_offset(const int g,
     const auto comp_idx
             = get_comp_ker_idx(kd_b, kd_e, kh_b, kh_e, kw_b, kw_e, cur_comp_oh);
     assert(IMPLICATION(jcp.req_cal_comp_pad, comp_idx >= 0));
-    return jcp.req_cal_comp_pad ? g * comp_ocb_sz + ocb * comp_ker_sz
-                    + comp_idx * comp_kw_sz + ow * comp_ow_sz
+    const int req_cal_comp_offset = g * static_cast<int>(comp_ocb_sz)
+            + ocb * static_cast<int>(comp_ker_sz)
+            + comp_idx * static_cast<int>(comp_kw_sz)
+            + ow * static_cast<int>(comp_ow_sz);
+    return jcp.req_cal_comp_pad ? req_cal_comp_offset
```

Comment:
I'm a little confused by the description  that you cast `int64_t` to `int` here to avoid int overflow? Are we sure that `req_cal_comp_offset` cannot exceed INT32_MAX?

Reply:
Sorry, this issue look false positive. I removed from PR.